Source/Android/jni/MainAndroid.cpp

```diff
@@ -604,6 +606,31 @@ JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_ReloadConfig
   SConfig::GetInstance().LoadSettings();
 }
 
+JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_SetOverlaySIDevice(
+    JNIEnv* env, jobject obj, jint gcController)
+{
+  SerialInterface::SIDevices SIDevice;
```

Comment:
```suggestion
  SerialInterface::SIDevices si_device;
```

We don't use camelCase variable naming in C++. (Well, at least not in newly written code... There is a bunch of legacy code still.)